react-hs-examples/redux/README.md

```diff
@@ -0,0 +1,13 @@
+redux
+=====
+
+Example react-hs application that uses redux architecture for managing state.
```

Comment:
"... uses redux-like architecture..." (to avoid confusion about library vs. idea)